### PR TITLE
New cmdlet Get-PnPTeamsPrimaryChannel to get primary channel

### PR DESCRIPTION
**Pull request by reshmee011**
## Type ##
- [ ] New Feature

## What is in this Pull Request ? ##
New cmdlet Get-PnPTeamsPrimaryChannel to get primary channel
Return property webUrl , email and createddatetime for channel

### CODE REVIEW

**Comment by KoenZomers** [comment]
Thanks @reshmee011 for your PR! Could you please also add a MD documentation file for this new cmdlet to your PR?

File: src/Commands/Teams/GetTeamsPrimaryChannel.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+using PnP.PowerShell.Commands.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace PnP.PowerShell.Commands.Teams
+{
+    [Cmdlet(VerbsCommon.Get, "PnPTeamsPrimaryChannel")]
+    [RequiredMinimalApiPermissions("Group.Read.All")]
```
Comment (KoenZomers) [inline src/Commands/Teams/GetTeamsPrimaryChannel.cs:14]: This permission seems not to be correct. See https://docs.microsoft.com/en-us/graph/api/team-get-primarychannel?view=graph-rest-1.0&tabs=http#permissions. I believe it should be Channel.ReadBasic.All.

Comment (reshmee011) [reply src/Commands/Teams/GetTeamsPrimaryChannel.cs]: Thanks @KoenZomers : I have updated the RequiredMinimalApiPermissions to Channel.ReadBasic.All.

**Comment by reshmee011** [comment]
@KoenZomers I have added the documentation for the cmdlet Get-PnPTeamsPrimaryChannel.

**Comment by KoenZomers** [comment]
Looks excellent @reshmee011, thanks for your contribtion.

**Pull request merged**